### PR TITLE
231 adj stock price need to consider capital reduction

### PR DESCRIPTION
**Pull request by machineCYC**
consider the reduction capital in taiwan_stock_daily_adj api

### CODE REVIEW

File: FinMind/data/data_loader.py [hunk 1]
```diff
@@ -94,6 +94,112 @@ def taiwan_stock_daily_adj(
...
+                        stock_price.loc[index - 1, "reduction_capital_close"]
+                        * stock_price.loc[index, "change"]
+                        + stock_price.loc[index - 1, "reduction_capital_close"]
+                    )
+                else:
+                    stock_price.loc[
+                        index, "reduction_capital_close"
+                    ] = stock_price.loc[index, "tmp_close"]
+
+            del stock_price["ReductionCapitalRatio"]
```
Comment (linsamtw) [inline FinMind/data/data_loader.py:160]: 怎麼不用 drop?

File: FinMind/data/data_loader.py [hunk 2]
```diff
@@ -94,6 +94,112 @@ def taiwan_stock_daily_adj(
...
+            stock_price["open"] = stock_price["reduction_capital_open"].round(2)
+            stock_price["max"] = stock_price["reduction_capital_max"].round(2)
+            stock_price["min"] = stock_price["reduction_capital_min"].round(2)
+            first_spread = stock_price["spread"].values[0]
+            stock_price["spread"] = stock_price["close"] - stock_price[
+                "close"
+            ].shift(1)
+            stock_price["spread"] = stock_price["spread"].fillna(first_spread)
+
+            del stock_price["reduction_capital_close"]
```
Comment (linsamtw) [inline FinMind/data/data_loader.py:198]: 怎麼不用 drop?

**Review by linsamtw — APPROVED** [review]

File: FinMind/data/data_loader.py (hunk 1 above)
Comment (machineCYC) [reply FinMind/data/data_loader.py]: update it!

File: FinMind/data/data_loader.py (hunk 2 above)
Comment (machineCYC) [reply FinMind/data/data_loader.py]: update it!

**Review by Featasy — APPROVED** [review]

**Pull request merged**